### ISSUE TITLE
Add Optimizely Integration to iOS Tracker

### ISSUE THREAD

**Issue by keanerobinson** [snowplow/snowplow#3620]
It would be awesome if we could build something similar to the Optimizely integration offered by the JS tracker for iOS. The integration could surface the same fields that are used in the `optimizelySummary` context ([link](https://github.com/snowplow/snowplow/wiki/1-General-parameters-for-the-Javascript-tracker#221411-optimizelysummary-context).

**Comment by alexanderdean** [comment]
Yes, this would be cool. I thought we had a ticket for this already, but can't find it...

**Comment by keanerobinson** [comment]
Apologies if there already is one - I couldn't find it! 

I'm working on getting this live on iOS with Lifecake next week. I'll see if there's anything we can use that Filippo builds!
